src/common/os_input_output.rs

```diff
@@ -268,10 +268,23 @@ impl OsApi for OsInputOutput {
     fn load_palette(&self) -> Palette {
         let palette = match Colors::new("xresources") {
             Some(palette) => {
-                let fg = hex_to_rgb(&palette.fg);
-                let bg = hex_to_rgb(&palette.bg);
-                let colors: Vec<(u8, u8, u8)> =
-                    palette.colors.iter().map(|c| hex_to_rgb(c)).collect();
+                let fg = if let Some(foreground) = hex_to_rgb(&palette.fg) {
```

Comment:
If `hex_to_rgb` should be like [this](https://github.com/zellij-org/zellij/pull/437/files#r624784691) comment, this line might be fixed with the following change (`palette.bg` is also fixed as the same change :smile:)

```suggestion
                let fg = if let Some(foreground) = palette.fg.as_deref().map(hex_to_rgb) {
```

Reply:
Yes, that makes sense :+1: .